### PR TITLE
[chains] Transcription Guide/Docs.

### PR DESCRIPTION
**Pull request by marius-baseten**
## :rocket: What
Also some renamings for more generality

### CODE REVIEW

File: docs/chains/example-transcribe.mdx [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+the chains framework makes internally). Furthermore, we need to take the
+invocation URL of that model (e.g.
+`https://model-5woz91z3.api.baseten.co/production/predict`) and pass it
+along when initializing this adapter class with the `from_url` factory-method.
+
+There are two options for deploying a model separately from a chain:
+
+#### As a Chainlet
+
+(not done in this example)
```
Comment (squidarth) [inline docs/chains/example-transcribe.mdx:227]: maybe spell out it the pros/cons of each as well

File: docs/chains/example-transcribe.mdx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  deployment and can auto-scale, depending on the load.
+* Once all tasks are dispatched, it waits for the results and concatenates
+  all the partial transcriptions from the chunks to a final output.
+
+### `MacroChunkWorker`
+
+The `MacroChunkWorker` chainlet works on chunk tasks it receives from the
+`Transcribe` chainlet. For each chunk it does the following:
+
+* It starts a `DownloadSubprocess` asynchronously (i.e. this will need CPU on
```
Comment (squidarth) [inline docs/chains/example-transcribe.mdx:177]: I think breaking this up some code samples could be good -- this is quite a bit of text. It also might be a bit more detail than the reader needs

File: docs/chains/example-transcribe.mdx [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+[model library](https://www.baseten.co/library/), the
+[truss examples repo](https://github.com/basetenlabs/truss-examples) or your
+[own truss model](https://truss.baseten.co/quickstart).
+
+This might be the better choice, if the model has a substantial code base
+itself and if you want to avoid mixing that (and the development of it) with
+the chain code.
+
+# Performance Considerations
+
```
Comment (squidarth) [inline docs/chains/example-transcribe.mdx:254]: Let's remind users where the full code for the example is again here

**Review by squidarth — APPROVED** [review]
Overall LGMT! My comments are that we might be going a little too deep into the weeds with the details of how to do this chunking, and that some more code samples could help spell things out

File: docs/chains/example-transcribe.mdx [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+</Tip>
+
+### `Transcribe`
+
+This chainlet is the "entrypoint" to the chain, external client send
+transcription requests to it. It's endpoint implementation has the following
+signature:
+
+```python
+async def run_remote(
```
[the remark this answers is not in the document]
Comment (marius-baseten) [reply docs/chains/example-transcribe.mdx:84]: There are examples shown in `with JSON input` code box.

**Pull request merged**